db/migrate/20161211162552_add_generating_ems_to_event_streams.rb

```diff
@@ -0,0 +1,5 @@
+class AddGeneratingEmsToEventStreams < ActiveRecord::Migration[5.0]
+  def change
+    add_column :event_streams, :generating_ems_id, :bigint, :after => :ems_id
```

Comment:
The :after didn't work for me, generating_ems_id was added at the end and I could never get it to work previously. According to Postgres wiki this has never been implemented in postgres

https://wiki.postgresql.org/wiki/Alter_column_position 

I'm Removing it for now.